### PR TITLE
Cber/switch for nginx metrics

### PR DESCRIPTION
**Pull request by ChristianBergen**
Adds a switch to be able to disable the metrics in the nginx_values.yaml

### CODE REVIEW

**Comment by schwichti** [comment]
Can you elaborate a bit why we need a switch for the metrics?

**Comment by amarin-dspace** [comment]
Maybe it's for the best, since we currently depend on simphera-monitoring helm chart to deploy necessary CRDs, in order to have metrics enabled and deployed with nginx helm chart.  
There is PR in simphera admin manual explaining the dependency of the two helm charts, but in order to avoid failed deployments (by default), I think this PR makes a lot of sense.

**Comment by ChristianBergen** [comment]
Yes, the case is, that our infrastructure creations fail because of the "metrics enabled" values in the nginx_values.yaml. By default they would still be enabled, but we as SystemTeam do not use this feature currently.

File: templates/nginx_values.yaml [hunk 1]
```diff
@@ -9,10 +9,6 @@ controller:
       service.beta.kubernetes.io/aws-load-balancer-target-node-labels: kubernetes.io/os=linux
       service.beta.kubernetes.io/aws-load-balancer-type: "nlb"
   metrics:
-    enabled: true
-    serviceMonitor:
-      enabled: true
-      additionalLabels:
-        release: "prometheus" 
+    enabled: false
```
Comment (schwichti) [inline templates/nginx_values.yaml:12]: @amarin-dspace I cannot assess if this change will break something

Comment (ChristianBergen) [reply templates/nginx_values.yaml:12]: In this case to have it configurable by a variable would be the better variant

Comment (ChristianBergen) [reply templates/nginx_values.yaml]: Reverted the last commit and set default value to 'false' in the variable.

**Review by amarin-dspace — APPROVED** [review]

**Comment by amarin-dspace** [comment]
LGTM, as we agreed in discussion with SimonS, we'll abandon these changes in TF and move nginx metrics' part completely to Simphera monitoring chart.

**Pull request merged**